src/master-playlist-controller.js

```diff
@@ -1122,25 +1131,30 @@ export class MasterPlaylistController extends videojs.EventTarget {
   }
 
   /**
-   * Exclude a playlist when an error occurs for a set amount of time
-   * making it unavailable for selection by the rendition selection algorithm
-   * and then forces a new playlist (rendition) selection.
+   * Exclude a playlist for a set amount of time, making it unavailable for selection by
+   * the rendition selection algorithm, then force a new playlist (rendition) selection.
    *
-   * @param {Object=} error an optional error that may include the playlist
-   * to exclude
-   * @param {number=} playlistExclusionDuration an optional number of seconds to exclude the
-   * playlist
+   * @param {Object=} playlistToExclude
+   *                  the playlist to exclude, defaults to the currently selected playlist
+   * @param {Object=} error
+   *                  an optional error
+   * @param {number=} playlistExclusionDuration
+   *                  an optional number of seconds to exclude the playlist
    */
-  excludePlaylist(error = {}, playlistExclusionDuration) {
+  excludePlaylist({
+    playlistToExclude = this.masterPlaylistLoader_.media(),
+    error = {},
+    playlistExclusionDuration
+  }) {
     // If the `error` was generated by the playlist loader, it will contain
     // the playlist we were trying to load (but failed) and that should be
     // excluded instead of the currently selected playlist which is likely
     // out-of-date in this scenario
```

Comment:
is this comment still correct? At the very least, we won't be using the error.playlist here anymore.

Reply:
Nice catch! I'll remove the whole comment.